cookbooks/ey-mysql/resources/mysql_slave.rb

```diff
@@ -106,6 +106,13 @@
     only_if { !Dir.glob("#{node['mysql']['datadir']}/slave-relay*").empty? && !volume_from_slave_snapshot }
   end
 
+  # the master writes it's uuid to <datadir>/auto.cnf, the slave needs that removed so it will gen it's own
+  file "#{node['mysql']['datadir']}/auto.cnf" do
+     action :delete
+     Chef::Log.info("removing <datadir>/auto.cnf")
+     only_if { node['mysql']['short_version'] >= '5.6' }   
```

Comment:
Looks good other than this line for linting

Should be `node["mysql"]["short_version"] >= "5.6"`

Reply:
Thank you, @mushyy This is now updated